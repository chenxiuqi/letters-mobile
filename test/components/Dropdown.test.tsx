import React from "react";
import { Dropdown } from "@components";
import renderer from "react-test-renderer";

const setup = () => {
  const element = renderer.create(<Dropdown />);
  const instance = element.getInstance();
  return {
    element,
    instance,
  };
};

describe("Dropdown component", () => {
  it("should render", () => {
    const { element } = setup();
    const tree = element.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
