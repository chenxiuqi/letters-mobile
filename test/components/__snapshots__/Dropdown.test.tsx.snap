// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown component should render 1`] = `null`;
